eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/processors/SubmittedVariantDeclusterProcessor.java

```diff
@@ -87,28 +91,26 @@ private boolean isSameType(DbsnpClusteredVariantEntity clusteredVariant, Submitt
                                 .equals(clusteredVariant.getType());
```

Comment:
Could you wrap this call with a try - catch of `IllegalArgumentException` and return false in that case? We agreed offline to consider that a type mismatch.